Index: ../../.gitignore
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.BaseRevisionTextPatchEP
<+>target/\nmyenv/\n**/.venv/\n**/.vscode\n**/__pycache__/\n**/expense_manager.db\n
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
diff --git a/../../.gitignore b/../../.gitignore
--- a/../../.gitignore	(revision 8f50e1a67573bb7114bc803477fcf46dcfad3ae0)
+++ b/../../.gitignore	(date 1767035875356)
@@ -4,3 +4,5 @@
 **/.vscode
 **/__pycache__/
 **/expense_manager.db
+**/.idea/
+**/.DS_Store
